### ISSUE TITLE
fix(model): keep Q2_K CPU prefill continuation logits finite

### ISSUE THREAD

**Issue by anthony-chaudhary** [anthony-chaudhary/fak#12433]
Parent context: #12038
Blocked leaf: #12346

## Classification
- Centrality: Core
- Priority tier: native serving
- Scope: Qwen3.5 dense CPU prefill/step with a packed Q2_K embedding; no GDN/tiled-convolution changes.

## For / Problem / Today / Better because / Witness
- **For:** native Qwen3.5 CPU sessions using packed Q2_K token embeddings.
- **Problem:** the committed continuation test produces non-finite logits before decode can continue.
- **Today:** at clean committed `ae2151e06`, `TestQwen35PrefillAndStepCPUContinuation` fails immediately with `Prefill logits[0] is not finite: NaN`. Its config has no `LayerTypes`, so it cannot enter the tiled GDN path being corrected under #12038.
- **Better because:** packed Q2_K prefill can hand a finite, valid logit vector to subsequent CPU decode steps without weakening numerical gates.
- **Witness:** from an isolated committed checkout, `go test -count=1 -run "^TestQwen35PrefillAndStepCPUContinuation$" ./internal/model` passes and all logits remain finite through prefill plus three steps.

## Reproduction
```text
go test -count=1 -run "^TestQwen35PrefillAndStepCPUContinuation$" ./internal/model
embedding_q2k_test.go:488: Prefill logits[0] is not finite: NaN
```

## Boundary
Investigate the Q2_K embedding/dense CPU quantized prefill path independently. Do not change tiled SiLU rounding, global tolerances, or the preserved two-file #12346 sampled-MTP patch.

**Comment by anthony-chaudhary** [comment]
Root cause and candidate receipt: this is an invalid synthetic fixture, not a production Q2_K dequantization defect. `TestQwen35PrefillAndStepCPUContinuation` omitted `RopeTheta`, so inverse frequencies for j>0 became +Inf and position-zero RoPE evaluated `0*Inf` to NaN. The one-file candidate adds the ordinary `RopeTheta: 10000` fixture field only.

Red-before at ae2151e06: `Prefill logits[0] is not finite: NaN`. Green-after: the exact finite prefill+three-step witness passes; it also passes inside the four-test #12038 blocker set and the full model+gateway run. No production, tolerance, compute, or #12346 file changed.

Guarded explicit-path commit refused before staging with `COMMITTED_RED` only because the automatic Strix selector classifies every `internal/model` test as device-relevant and the appliance probe returned SSH exit 255. That selector defect is #12382. No bypass flag was used, and no commit/push exists. Keep #12433 open until a synchronized guarded landing produces a changed `internal/model@rev`.
